### ISSUE TITLE
Issue #95 --> address found issues in rst files

### ISSUE THREAD

**Issue by lindseyc** [google/applied-computing-series#95]
While converting into markdown, found some areas where there was no alt text, missing questions, private links and no formatting on images.
